docs/api/en/materials/Material.html

```diff
@@ -175,13 +175,6 @@ <h3>[property:Float opacity]</h3>
 		Default is *1.0*.
 		</p>
 
-		<h3>[property:Float overdraw]</h3>
-		<p>
-		Amount of triangle expansion at draw time.
-		This is a workaround for cases when gaps appear between triangles when using [page:CanvasRenderer].
-		*0.5* tends to give good results across browsers. Default is *0*.
-		</p>
-
```

Comment:
We should also remove: https://threejs.org/docs/index.html#api/zh/materials/Material.overdraw

Reply:
I do not think 'we' should do that. That is the responsibility of the maintainers of those docs.